doc/man7/migration_guide.pod

```diff
@@ -121,6 +121,27 @@ FIPS module, as detailed below. Authors and maintainers of external engines are
 strongly encouraged to refactor their code transforming engines into providers
 using the new Provider API and avoiding deprecated methods.
 
+=head3 Support of legacy engines
+
+If openssl is not built without engine support or deprecated API support, engines
+will still work. However, their applicability will be limited.
+
+New algorithms provided via engines will still work.
+
+Engine-backed keys can be loaded via custom B<OSSL_STORE> implementation.
+In this case the B<EVP_PKEY> objects created via L<ENGINE_load_private_key(3)>
+will be concidered legacy and will continue to work.
+
+Engine-based hardware acceleration for asymmetric algorithms will be available
+if you use the legacy API (e.g. B<RSA_METHOD>) and create the RSA object from
+parameters.  However, if you get the B<RSA> object with
+L<PEM_read_bio_PrivateKey(3)> and L<EVP_PKEY_get1_RSA(3)>, the hardware offload will
```

Comment:
This is not correct. Please see my latest post in #17081

Reply:
Fixed. Thanks for your investigation!